include/bar.hpp

```diff
@@ -58,8 +58,9 @@ class Bar {
   Bar(const Bar &) = delete;
   ~Bar() = default;
 
-  void setVisible(bool visible);
-  void toggle();
+  auto setVisible(bool nvis) -> void;
+  auto toggle() -> void;
```

Comment:
I know I'm nitpicking, but why was that change necessary?

Reply:
No that's totally justified. I took the wrong definition while resolving the merge conflicts. I will correct this.